src/renderer/containers/Console/index.tsx

```diff
@@ -200,3 +202,33 @@ const MirroringHelp = () => {
     </>
   );
 };
+
+const OBSWebsocketNotice = () => {
```

Comment:
Can you move this into it's own component, and then within that same file/component just declare:

```typescript
const OBS_WEBSOCKET_NOTICE_KEY = "SEEN_OBS_WEBSOCKET_NOTICE";

// Inside the component
const [seenNotice, setSeenNotice] = React.useState<boolean>(localStorage.getItem(OBS_WEBSOCKET_NOTICE_KEY) === "true");
const onClose = () => {
  localStorage.setItem(OBS_WEBSOCKET_NOTICE_KEY, "true");
  setSeenNotice(true);
};
```

And then revert `useAdvancedUser` back to how it was? This local storage key is only ever going to be used within this component where as the other one is used in a few places.